docs/content/en/docs/reference/clusterspec/privateregistry.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Private Registry configuration"
+linkTitle: "Private Registry"
+weight: 90
+description: >
+  EKS Anywhere cluster yaml specification private registry configuration reference
+---
+
+## Private Repository Support (optional)
+You can configure EKS Anywhere to use a private registry. The following cluster spec 
+shows an example of how to configure private registries:
+```yaml
+apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+kind: Cluster
+metadata:
+   name: my-cluster-name
+spec:
+   ...
+  registryMirrorConfiguration:
+    endpoint: <private registry IP/hostname address>
+    CACertContent: CA certificate
+```
+## Private Registry Configuration Spec Details
+### __registryMirrorConfiguration__ (required)
+* __Description__: top level key; required to use a private registry.
+* __Type__: object
+
+### __endpoint__ (required)
+* __Description__: IP address or hostname of the private registry to be used when pulling images during cluster creation 
+* __Type__: string
+* __Example__: ```endpoint: 192.168.0.1```
+### __CACertContent__ (optional)
+* __Description__: Certificate Authority's Certificate used for TLS. When using 
+  self-signed certificates it is necessary to pass this parameter in the cluster spec.
+  It is also possible to configure CACertContent by exporting an environment variable:
+  export EKSA_REGISTRY_MIRROR_CA="/path/to/certificate-file"
+* __Type__: string
+* __Example__: <br/>
+  CACertContent: |<br/>
+  &nbsp;&nbsp;&nbsp;&nbsp;-----BEGIN CERTIFICATE-----<br/>
+  &nbsp;&nbsp;&nbsp;&nbsp;AbCdEfG........................................<br/>
+  &nbsp;&nbsp;&nbsp;&nbsp;-----END CERTIFICATE-----<br/>
+
```

Comment:
I don't know what is planned here, but I feel there should be a chunk in the tasks section about this with the content after this comment and expanded out a bit. It should be similar to https://anywhere.eks.amazonaws.com/docs/tasks/cluster/cluster-flux/ with a link to this reference. Generally, looks good!